activate-virtualenv.sh

```diff
@@ -24,17 +24,17 @@ if ! type "workon" &>/dev/null; then
   source $virtualenvwrapper_path
 fi
 
-if ! which python2.7 &>/dev/null; then
-  echo 'Error: python2.7 is not in your path.' \
-       'Please ensure python2.7 is installed and available'
+if ! which python3.6 &>/dev/null; then
+  echo 'Error: python3.6 is not in your path.' \
+       'Please ensure python3.6 is installed and available'
```

Comment:
Do we maybe want to link to https://github.com/cfpb/development/blob/master/guides/installing-python.md here? This didn't exist when this script was created, but it might be a nice addition.

Reply:
Great suggestion. I added the link you recommended to activate-virtualenv.sh